test-files/cmd/quick/ConvertGeoNames.sh.stdout

```diff
@@ -1,10 +1,15 @@
-20:16:22.637 INFO  ...hoot/core/io/DataConverter.cpp( 184) Converting test-files/cmd/quick/ConvertGeoNames.geonames to /dev/stdout...
-20:16:22.640 INFO  ...ot/core/io/ElementStreamer.cpp( 132) Unable to stream I/O due to: hoot::TranslationOp
-20:16:22.642 INFO  ...ore/io/OsmMapReaderFactory.cpp( 168) Loading map from test-files/cmd/quick/ConvertGeoNames.geonames...
-20:16:22.644 INFO  ...chema/JavaScriptTranslator.cpp( 239) Loading translation script: /home/vagrant/hoot/translations/GeoNames.js
-20:16:22.657 INFO  ...chema/JavaScriptTranslator.cpp( 239) Loading translation script: /home/vagrant/hoot/translations/GeoNames.js
-20:16:22.665 INFO  ...chema/JavaScriptTranslator.cpp( 239) Loading translation script: /home/vagrant/hoot/translations/GeoNames.js
-20:16:22.674 INFO  ...ore/io/OsmMapWriterFactory.cpp( 129) Writing map to /dev/stdout...
+09:10:40.001 STATUS ...pp/hoot/core/util/Progress.cpp(  85) Convert (0%): Converting ...quick/ConvertGeoNames.geonames to .../dev/stdout...
```

Comment:
Change this test to run with `--warn`. Too much stuff in the log. All we really need to see is that the diff is empty.

Reply:
I added `--warn` but the log size doesn't decrease.  The two convert commands write to stdout

Reply:
I didn't see that part. That test was set up weird from the beginning I guess. Remove the write to stdout, add warn level logging, and add a diff between a gold file and the output file for each command call. That's how tests of this type are configured.

Reply:
Done.

Reply:
awesome. thanks.